### PR TITLE
all: use jsonhttp

### PR DESCRIPTION
**Pull request by rogpeppe**
Also sort imports in the standard way.

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/364/

**Comment by bac** [comment]
:+1:

File: internal/router/util.go [hunk 1]
```diff
@@ -66,6 +53,14 @@ func ErrorResponse(err error) *params.Error {
 	return errResp
 }
 
+type errorInfoer interface {
```
Comment (urosj) [inline internal/router/util.go:56]: Please add a // todo here to resolve the name later, so that it represents that it collects multiple errors.

**Comment by urosj** [comment]
:+1: with a small comment.

**Comment by rogpeppe** [comment]
:shipit:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/373/

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**